### DISCUSSION TITLE
Noobie Needs Help!

### DISCUSSION THREAD

**Discussion by ukwilky** [HandyHat/ha-hildebrandglow-dcc#299] [General]
Hi,

I have only had Home Assistant for a few days and have jumped in to custom integrations to get information from my SMETS2 meter via DCC with the Bright App.

I have added the custom config now twice and it still doesn't show within the integration pane.

I did restart HA. Do you have any ideas, why this wou;dn't show?

Thanks,

ukwilky

**Comment by HandyHat** [comment]
Hey @ukwilky, are you using HACS?
If you are manually installing the component, make sure you copy the entire directory to the `custom_components` folder.
If you've installed it correctly and restarted, the My Home Assistant link (https://my.home-assistant.io/redirect/config_flow_start/?domain=hildebrandglow_dcc) should work.

If you are trying to manually find the integration in the list, make sure you've cleared your browser cache.

**Reply by ukwilky** [reply]
Hey @HandyHat,

Damn, I forgot to mention that I was using HACS, to install this. 

I just tried the link above and got the following Error:

This integration does not support configuration via the UI. If you followed this link from the Home Assistant website, make sure you run the latest version of Home Assistant.

I just checked and I am running the latest: 

Home Assistant 2023.1.6
Supervisor 2022.12.1 Operating System 9.4

Shall I just try the manual method?

Thanks,

Ian

**Reply by HandyHat** [reply]
Hey @ukwilky 

That error sounds like the integration isn't installed properly. To confirm, you added the custom repository via HACS and then installed the integration?

I would give HACS a go again and if it still doesn't work then you can give the manual method a try

**Reply by ukwilky** [reply]
Hey @HandyHat 

Thanks for coming back to me. Yes. I added the custom repository via HACS. 

I will remove HACS and reinstall and try again. If not, I will take your advice and give the manual method a try.
